screens/Media.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
`useRef` isn't used

Reply:
But I wanted it to be used, ok, @dereckquock ?! I'll remove it 😞